src/soundio/sounddevicenetwork.h

```diff
@@ -85,6 +89,16 @@ class SoundDeviceNetworkThread : public QThread {
 
   private:
     void run() {
+
+#ifdef __LINUX__
```

Comment:
Please add a comment for why this is only necessary on Linux.

Reply:
I have no idea how the situation is on the other OSs alt least portaudio has this feature for ALSA only. 
Who can verify this on windows an Mac Os?
Is there a tool to visualise thread priorities on windows?